packages/rancher-logging/generated-changes/overlay/templates/loggings/root/logging.yaml

```diff
@@ -17,7 +18,11 @@ spec:
       roleBasedAccessControlCreate: true
     {{- end }}
   {{- if .Values.global.dockerRootDirectory }}
-    mountPath: {{ .Values.global.dockerRootDirectory }}
+    mountPath: {{ $containers }}
+    extraVolumeMounts:
+      - source: {{ $containers }}
+        destination: {{ $containers }}
+        readOnly: true
```

Comment:
Looks like `extraVolumeMounts` is a banzai chart thing, just another way of specifying mounts ?

Reply:
Yes. This is required for the symlink to work though.